### ISSUE TITLE
wp8.1 does not show banners

### ISSUE THREAD

**Issue by jenokizm** [Adrotator/AdrotatorV2#69]
I place under the banner blank. No error appears, but also do not see the banner (
Code:
~
<adRotator:AdRotatorControl 
        x:Name="MyAdRotatorControl"
        AdHeight="90"
        AdWidth="728"
        LocalSettingsLocation="defaultAdSettings.xml"

```
    AutoStartAds="True" />
```

<?xml version="1.0" encoding="utf-8"?>
<AdSettings xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
    <CultureDescriptors CultureName="">
    <AdDuplex AppId="1379" Probability="10"/>
    <AdMob AppId="ca-app-pub-2213751212440288/6666168142" Probability="10" AdOrder="1"/>
    </CultureDescriptors>
</AdSettings>
~

**Comment by SimonDarksideJ** [comment]
As a simple test.
Can you add the providers XAML control to your page with the ad provider ID's you have defined in your AdSettings.xml.  
We ask this to check if the adprovider is actually sending ads for your ID's as they are not 100% guaranteed to send an ad

**Comment by jenokizm** [comment]
Maybe that that I need to manually install the library providers. I need only admob. But the reference in readme see on the SDK for ios(
How do I need to download the library, it is certainly to your component displays advertisements admob?

**Comment by SimonDarksideJ** [comment]
Yes, for AdRotator to function you must install the original Ad Provider framework DLL's.  AdRotator simply provides a shell around all the installed AdProvider frameworks, managing fill rates and rotation through them.
AdRotator (depending on the Windows platform you are running on) supports MS Ads, AdMob, Smaato, InMobi, MobFox, etc (details on the getadrotator.com site)

**Comment by jenokizm** [comment]
Well, that's understandable. I also saw the site. However, it is unclear exactly where to get the library to admob wp8.1
Do not share a direct link to dll?

**Comment by SimonDarksideJ** [comment]
Sadly Google do not have an 8.1 Dll last time I checked.  
You can check the google doc's here
https://developers.google.com/admob/wp/quick-start

Although looking for those docs was a lot harder than the last time I looked,  Google really don't like MS :S
